paasta_tools/slack.py

```diff
@@ -72,3 +73,18 @@ def get_slack_client():
     if token is None:
         token = optionally_load_system_paasta_config().get_slack_token()
     return PaastaSlackClient(token=token)
+
+
+def get_authors_to_be_notified(git_url, from_sha, to_sha):
```

Comment:
I'm not convinced this is better off here than in mark_for_deployment.py.

Reply:
Ok